pkgs/top-level/config.nix

```diff
@@ -117,28 +124,105 @@ let
       '';
     };
 
-    showDerivationWarnings = mkOption {
-      type = types.listOf (types.enum [ "maintainerless" ]);
-      default = [];
-      description = lib.mdDoc ''
-        Which warnings to display for potentially dangerous
-        or deprecated values passed into `stdenv.mkDerivation`.
-
-        A list of warnings can be found in
-        [/pkgs/stdenv/generic/check-meta.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/check-meta.nix).
-
-        This is not a stable interface; warnings may be added, changed
-        or removed without prior notice.
-      '';
-    };
-
     checkMeta = mkOption {
       type = types.bool;
-      default = false;
+      default = true;
```

Comment:
I don't remember the RFC saying that meta would be checked by default. This seems to cause a considerable increase in resource usage: https://github.com/NixOS/nixpkgs/pull/177272/checks?check_run_id=15236964430 (8.50% increase in memory usage), which I don't think we can justify

Reply:
You apparently [tried this before](https://github.com/NixOS/nixpkgs/pull/191171#issuecomment-1260692375) but it [was reverted](https://github.com/NixOS/nixpkgs/pull/194058).

Reply:
Oh I'm sorry, this is still in there from testing. I have no aspirations to enabling this flag again given the backlash last time ^^